app/models/concerns/wranglable.rb

```diff
@@ -0,0 +1,16 @@
+module Wranglable
+  extend ActiveSupport::Concern
+
+  included do
+    after_save :update_last_wrangling_activity
+    after_destroy :update_last_wrangling_activity
+  end
+
+  def update_last_wrangling_activity
+    current_user = User.current_user
+    return unless current_user&.respond_to?(:is_tag_wrangler?) && current_user&.is_tag_wrangler?
+
+    last_activity = LastWranglingActivity.find_or_create_by(user: User.current_user)
```

Comment:
Could this be just `user: current_user`, since we already defined the variable on line 10?